project/src/test/java/com/google/sps/VolunteerOpportunityTest.java

```diff
@@ -4,13 +4,13 @@
 import com.google.sps.data.VolunteeringOpportunity;
```

Comment:
Would you mind removing these changes to VolunteerOpportunityTest.java? I think they are doing formatting that will cause the presubmit to fail.

Reply:
My linter passes, I think I just did this so the linter would pass, but feel free to change in other PRs.